### PR TITLE
ROC-7377 | added new state and event

### PR DESCRIPTION
**Pull request by akritis22**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-7607

### Change description ###
1) A new event **CaseProceedsInCaseman** added
2) A new state **judgmentRequested** added
3) Changed PostConditionState for **CCJRequested** event to **judgmentRequested**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: definition/data/sheets/State.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
...
   {"CaseTypeID": "MoneyClaimCase", "ID": "referredMediation", "Name": "Referred to mediation", "Description": "Mediation referred", "DisplayOrder": 15, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "proceedsInCaseman", "Name": "Proceeds in Caseman", "Description": "Rest of the journey will be proceeded in caseman", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "awaitingCitizenPayment", "Name": "Payment pending", "Description": "Payment pending", "DisplayOrder": 17, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "awaitingCitizenPayment", "Name": "Payment pending", "Description": "Payment pending", "DisplayOrder": 17, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "judgmentRequested", "Name": "Judgment Requested", "Description": "Judgment requested for ccj", "DisplayOrder": 18, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"}
```
Comment (andrewwa-kainos) [inline definition/data/sheets/State.json]: This description seems to have some redundancy. "Judgment requested" should be enough

Comment (akritis22) [reply definition/data/sheets/State.json]: @andrewwa-kainos done

File: definition/data/sheets/CaseEvent.json [hunk 2]
```diff
@@ -66,5 +66,6 @@
...
-  {"CaseTypeID": "MoneyClaimCase", "ID": "ResetPin", "Name": "Reset Pin", "Description": "Reset and send pin to defendant", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": "5,10,15"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ResetPin", "Name": "Reset Pin", "Description": "Reset and send pin to defendant", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": "5,10,15"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "CaseProceedsInCaseman", "Name": "Case proceeds in Caseman", "Description": "Case to continue in Caseman", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "proceedsInCaseman", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": "5,10,15"}
```
Comment (sabahirfan) [inline definition/data/sheets/CaseEvent.json]: Do we  need this callback here? if yes where is the implementation for this ?

Comment (dharmendrak) [reply definition/data/sheets/CaseEvent.json]: no callback sabah, just for caseworker to update the case state

Comment (andrewwa-kainos) [reply definition/data/sheets/CaseEvent.json]: So the callback properties should be removed?

Comment (akritis22) [reply definition/data/sheets/CaseEvent.json]: there will be a call back for the event which I am working on right now in claim store. So when the case worker triggers this event, the state of the case will be updated to judgmentRequested.

Comment (akritis22) [reply definition/data/sheets/CaseEvent.json]: that's removed now

**Pull request closed without merging**